src/main/java/com/star_trello/darkside/service/UserSessionService.java

```diff
@@ -56,13 +56,13 @@ public UserSession createSession(int id) {
     }
 
     @Transactional
-    public User getUserByToken(UserSession userSession) {
-        boolean isAuth = userSessionRepo.existsByToken(userSession.getToken());
+    public User getUserByToken(String token) {
+        boolean isAuth = userSessionRepo.existsByToken(token);
```

Comment:
inline this please) And we realy return null?

Reply:
Now we can close eyes on that. Later, we can do annotation `@CheckAuthorization(token)`, that will check token in db, and if token is valid, annotation would return current user. If token is invalid, the annotation would return 401 Error (not authorized)